website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -0,0 +1,120 @@
+---
+page_title: Manage projects in HCP Terraform and Terraform Enterprise
+description: |-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure.
+---
+
+# Manage projects
+
+This topic describes how to create and manage projects in HCP Terraform and Terraform Enterprise. A project is a folder containing one or more workspaces.
+
+## Requirements
+
+You must have the following permissions to manage projects:
+
+- You must be a member of a team with the **Manage all Projects**  permissions enabled to create a project. Refer to [Organization Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) for additional information.
+- You must be a member of a team with the update team access to add a new team access for organization-visible teams for a project.
```

Comment:
I'm not 100% if this is the permission being talking about but I *think* it is, so here is my best guess:
```suggestion
- You must be a member of a team with the [team management "manage" permission](/terraform/cloud-docs/users-teams-organizations/permissions#general-project-permissions) to allow non-secret teams to access a project.
```

Reply:
```suggestion
- You must be a member of a team with the **Visible** option enabled under **Visibility** in the organization settings to configure a new team's access to the project. Refer to [Team Visibility](/terraform/cloud-docs/users-teams-organizations/teams#team-visibility) for additional information.
```
Yeah, that wasn't really clear.